test/functional/http/get/transactions.js

```diff
@@ -383,6 +398,52 @@ describe('GET /api/transactions', () => {
 			});
 		});
 
+		describe('senderIdOrRecipientId', () => {
+			it('using invalid senderIdOrRecipientId should fail', () => {
+				return transactionsEndpoint
+					.makeRequest({ senderIdOrRecipientId: '' }, 400)
+					.then(res => {
+						expectSwaggerParamError(res, 'senderIdOrRecipientId');
+					});
+			});
```

Comment:
@Tschakki This case handles the case fo `empty` string. You should also add a case where you specify value but invalid value.